benchmark_runner/common/template_operations/templates/windows/internal_data/namespace_template.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ namespace }}
+
+
```

Comment:
There are a lot of identical templates for this; it would be good to have just one template for creating namespaces.  Not necessary to fix this now, but I've opened #574 against this.